### ISSUE TITLE
How should we handle "syntactical" edits that require a set of LARs?

### ISSUE THREAD

**Issue by hkeeler** [cfpb/hmda-platform#474]
As mentioned on https://github.com/cfpb/hmda-platform/pull/452#discussion_r73175739, `LarEngine.validateLars` is never used.  Why?  Likely because the only function it calls ([`LarSyntacticalEngine.checkSyntacticalCollection`](https://github.com/cfpb/hmda-platform/blob/master/validation/src/main/scala/hmda/validation/engine/lar/syntactical/LarSyntacticalEngine.scala#L20-L27)) deals with these two oddball edits:
- [S011](https://github.com/cfpb/hmda-platform/issues/92)
- [S040](https://github.com/cfpb/hmda-platform/issues/95)

They're labeled as _syntactical_, but I'd argue that's not really what they are.  They feel closer to _macro_ edits...sorta.

Long story short, these are _different_, and we'll likely need to handle them differently than they currently are.
